src/views/blocks/rsvp/content.php

```diff
@@ -8,23 +8,31 @@
  *
  * See more documentation about our Blocks Editor templating system.
  *
- * @link {INSERT_ARTICLE_LINK_HERE}
+ * @link    https://m.tri.be/1amp Help article for RSVP & Ticket template files.
  *
- * @since 4.9
- * @since 4.12.0 Add $post_id to filter for hiding opt-outs.
+ * @since   4.9
+ * @since   4.12.0 Add $post_id to filter for hiding opt-outs.
  *
  * @version 4.12.0
  *
+ * @var Tribe__Tickets__Editor__Template $this            The template instance.
+ * @var Tribe__Tickets__Ticket_Object    $ticket          The ticket object.
+ * @var int                              $threshold       The threshold value to show or hide quantity available.
+ * @var int                              $available_count The quantity of Available tickets based on the Attendees number.
+ * @var bool                             $show_unlimited  Whether to allow showing of "unlimited".
+ * @var bool                             $is_unlimited    Whether the ticket has unlimited quantity.
+ * @var int                              $post_id         The Post ID the RSVP is attached to.
  */
-$going = ! empty( $_GET[ 'going' ] ) ? sanitize_text_field( $_GET[ 'going' ] ) : '';
+$going = $this->get( 'get' );
+//$going = ! empty( $_GET[ 'going' ] ) ? sanitize_text_field( $_GET[ 'going' ] ) : '';
```

Comment:
* :warning: This comment is 56% valid code; is this commented out code?
* :no_entry_sign: No space found before comment text; expected "// $going = ! empty( $_GET[ 'going' ] ) ? sanitize_text_field( $_GET[ 'going' ] ) : '';" but found "//$going = ! empty( $_GET[ 'going' ] ) ? sanitize_text_field( $_GET[ 'going' ] ) : '';"

<sub>hash: a6b98664dbf98ad4b81712ce0f77a2d4</sub>